### PR TITLE
RD-6485 Add support for sorting deployment inputs and workflow parameters

### PR DESCRIPTION
**Pull request by qooban**
## Description

This PR adds sorting capabilities for:
* deployment inputs in Create Deployment modal
* workflow parameters in Execute Workflow modal

Additional changes:
* Fixed hooks issue (hook inside hook)
* Improved translation for common inputs
* Improved `withinAccordionSection` Cypress custom command 

### Deployment inputs sorting

https://user-images.githubusercontent.com/5202105/208092940-3af365e7-dc16-4d26-9925-d728a8c1801e.mp4

### Workflow parameters sorting

https://user-images.githubusercontent.com/5202105/208092952-d0aa62e0-5500-4ee6-99e5-36fcae41435c.mp4

## Checklist
- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
Improved Deployment Inputs spec and Workflow Parameters spec.
1. [Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/4142/) - full run, failed on modified specs
2. [Build Status](https://jenkins.cloudify.co/job/Stage-UI-System-Test/4146/) - after https://github.com/cloudify-cosmo/cloudify-stage/pull/2341/commits/3f6cac8537a485e671ed3d56afd4236a503b9f59, https://github.com/cloudify-cosmo/cloudify-stage/pull/2341/commits/55a7c0213a4280b8772cb3385b1ceb3fbef3aa5e and https://github.com/cloudify-cosmo/cloudify-stage/pull/2341/commits/133396263c69e88e2da960417a32629ed9372ae1, only on modified specs - 2/3 failed, but locally all are passing (I suspect environment instability - unrelated to my changes)
3. [Build Status](https://jenkins.cloudify.co/job/Stage-UI-System-Test/4147/) - PASSED (no changes in comparison to previous run, so IMHO it proved that previous failures were caused by instability)

## Documentation
Will be done as part of RD-6486.

### CODE REVIEW

File: app/widgets/common/inputs/DataTypesButton.tsx [hunk 1]
```diff
@@ -6,8 +6,7 @@ import ParameterValue from '../components/parameter/ParameterValue';
 import StageUtils from '../../../utils/stageUtils';
 import { AnyData } from '../../../utils/props';
 import { Button, CancelButton, Header, Modal, Popup, Segment, Table } from '../../../components/basic';
-
-const t = StageUtils.getT('widgets.common.deployments.deployModal');
+import translateInputs from './utils/tranlateInputs';
```
Comment (PatrycjaKubiczek) [inline app/widgets/common/inputs/DataTypesButton.tsx:9]: typo in file  `'./utils/tranlateInputs';` missing `s` :)

**Review by PatrycjaKubiczek — COMMENTED** [review]
Tested UI, looks really good :) 

**Optional:**
I'm wondering if sorting icons should be visible if there is only one input available for example here:
Zrzut ekranu 2022-12-16 131744

**Review by kubama — DISMISSED** [review]

File: app/widgets/common/inputs/DataTypesButton.tsx (hunk 1 above)
Comment (qooban) [reply app/widgets/common/inputs/DataTypesButton.tsx]: Fixed 🙂

**Comment by qooban** [comment]
> **Optional:**
> I'm wondering if sorting icons should be visible if there is only one input available for example here:

Good idea, changed to show them only when there's more than 1 input available 👍🏻

**Comment by qooban** [comment]
It looks like I messed something in added system tests, I'll fix them in the evening and leave this PR for final approval and potential merge in the next week.

**Comment by qooban** [comment]
Fixed modified specs. All tests passed.

@kubama @PatrycjaKubiczek Feel free to merge this PR in the next week. If you have any comments, I'll gladly address them in the new year.

**Review by kubama — APPROVED** [review]

**Pull request merged**